### ISSUE TITLE
[wg/did] Charter proposal: rechartering the Decentralized Identifier WG

### ISSUE THREAD

**Issue by pchampin** [w3c/strategy#376]
New charter proposal, reviewers please take note.

# Charter Review

Charter: https://w3c.github.io/did-wg-charter/

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

 - [x] Existing WG recharter 

- current charter at https://w3c.github.io/did-wg-charter/

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, and security. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).**

Communities suggested for outreach: 

* [Credentials CG](https://www.w3.org/groups/cg/credentials)

Known or potential areas of concern: 

* there is dissent in the current WG about the content of this charter proposal, which is explicitly documented in the introduction of  the charter. 

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)

* https://github.com/w3c/did-wg-charter/issues

Anything else we should think about as we review?

**Comment by pchampin** [comment]
@brentzundel

**Comment by himorin** [comment]
memo for HR groups
current running charter: https://www.w3.org/2020/12/did-wg-charter.html
diff: https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2020%2F12%2Fdid-wg-charter.html&doc2=https%3A%2F%2Fw3c.github.io%2Fdid-wg-charter%2F

**Comment by pchampin** [comment]
The following paragraph is currently in the charter, but @plh suggested that it should rather be (1) removed from the charter and (2) included in the email to the AC announcing the proposed charter:


>  We note that including the option to standardize DID Methods within the scope of  this charter was controversial within the existing DID WG. There was dissent in how best to address the director's advice prior to the publication of <a href="https://www.w3.org/TR/did-core/">Decentralized Identifiers (DIDs) v1.0</a>  as a Recommendation, which was to address and deliver proposed standard DID  Method(s) and demonstrate interoperable implementations during its next  chartered period.

**Comment by himorin** [comment]
No comment nor request from i18n

**Comment by brentzundel** [comment]
@matatk Though DIDs may be useful in some designs for replacing CAPTCHAs, I expect Verifiable Credentials to play a much more important role.

**Comment by matatk** [comment]
Thanks @brentzundel, that makes sense. We have a long-running thread with Verifiable Credentials, so will pick it up.

Meanwhile, I re-confirm that we're happy to sign off on your charter. @ruoxiran explained how I can do that by adjusting the labels, so I'll do that now.

**Comment by plehegar** [comment]
update: the Team is looking into a [potential concern](https://blog.joeandrieu.com/2023/04/12/fighting-for-consensus-at-the-w3c/)

**Comment by brentzundel** [comment]
I'm happy to meet with the team to discuss

**Comment by plehegar** [comment]
update: the Team is looking to gather more information from the DiD Working Group.

**Comment by plehegar** [comment]
The latest attempt at resolving the formal objections failed. We started with 4 formal objections, resolved 2, and received an additional objection yesterday.

A Council will need to look at the remaining formal objections.

**Comment by samuelweiler** [comment]
As background, [charter sent for AC review on 7 August 2023](https://lists.w3.org/Archives/Public/public-new-work/2023Aug/0002.html) (public)
[Call for AC review](https://lists.w3.org/Archives/Member/w3c-ac-members/2023JulSep/0016.html). (member-only)
[Charter snapshot](https://www.w3.org/2023/07/did-wg-charter.html) (public)
[Results](https://www.w3.org/2002/09/wbs/33280/did-wg-2023/results) (member-only)

Three public formal objections
[1](https://lists.w3.org/Archives/Public/public-new-work/2023Sep/0011.html)
[2](https://lists.w3.org/Archives/Public/public-new-work/2023Aug/0015.html)
[3](https://lists.w3.org/Archives/Public/public-new-work/2023Sep/0008.html)

**Comment by brentzundel** [comment]
It is my understanding that #2 from the list provided by Sam has been
resolved.

**Comment by plehegar** [comment]
1. @pchampin is going to write a report for the W3C Council summarizing the decision and the objection.
2. @ylafon is the Council Team Contact to support the Council

For details, see [referral to W3C Council](https://www.w3.org/Guide/council/council.html#referral).

**Comment by mnot** [comment]
@plehegar I can't tell from that link -- will there be an announcement of the report, and will there be an opportunity for input to the council by the AC and/or public?

**Comment by plehegar** [comment]
@mnot Whether a Council welcomes or not  information from the public is up to each Council. The Team Contact for each Council should make a Council aware when they receive information. As mentioned above, @ylafon is the Team Contact for the DiD Council.

**Comment by ylafon** [comment]
Note that the Team Report, if it can be shared, is added to the [Formal Objection Dashboard](https://www.w3.org/Member/wiki/DirectorFOdashboard) this is the place to track Formal Objections status.
The Council report is announced when the Council concludes.

**Comment by mnot** [comment]
Thanks @ylafon. That dashboard is very hard to find unless you're 'in the know' -- perhaps it could be linked from https://www.w3.org/Member (although that page is already very crowded and hard to navigate)?

**Comment by plehegar** [comment]
> Thanks @ylafon. That dashboard is very hard to find unless you're 'in the know' -- perhaps it could be linked from https://www.w3.org/Member (although that page is already very crowded and hard to navigate)?

@koalie ^

**Comment by plehegar** [comment]
Council requested[ clarification and a 28 days review period](https://lists.w3.org/Archives/Member/member-charters-review/2024Mar/0008.html). Deadline is 12 April 2024. 

Latest [proposed charter](https://www.w3.org/2024/03/proposed-wg-did.html).

**Comment by koalie** [comment]
> That dashboard is very hard to find unless you're 'in the know' -- perhaps it could be linked from https://www.w3.org/Member (although that page is already very crowded and hard to navigate)?

Hi @mnot I added a link to the formal objection dashboard in the right-hand side menu of the Member homepage.

**Comment by mnot** [comment]
Thank you @koalie!

**Comment by plehegar** [comment]
The latest  [proposed charter](https://www.w3.org/2024/03/proposed-wg-did.html) did not received objections and the Council was [dissolved](https://lists.w3.org/Archives/Member/w3c-ac-forum/2024AprJun/0032.html).

**Comment by cwilso** [comment]
Note this should be "did not receive Formal Objections".

**Comment by himorin** [comment]
quite minor points (per recent template change)

- remove `performance` from 5. Coordination 1st paragraph
- last at 6. Participation, `Code of Ethics and Professional Conduct` has been renamed as `Code of Conduct`
- 4th paragraph in 8. Decision Policy, remove director from the last `at the discretion of the Chairs or the Director. `

**Comment by plehegar** [comment]
Approved
https://www.w3.org/2024/04/did-wg-charter.html
